.circleci/config.yml

```diff
@@ -35,7 +26,13 @@ e2eTest: &e2eTest
     - run:
         name: Run test
         command: |
-          E2E_KUBECONFIG=/e2e/kubeconfig ./e2e-harness test --test-dir=${TEST_DIR}
+          docker run --rm --network host \
+            -v $(pwd)/${TEST_DIR}:/e2e \
+            -v $(pwd):/go/src/github.com/giantswarm/kubernetes-nginx-ingress-controller \
+            -e E2E_KUBECONFIG=/e2e/kubeconfig \
+            -e CIRCLE_SHA1=${CIRCLE_SHA1} \
```

Comment:
It would be cool to read project.yaml and pass those.

Reply:
Will apply same as you did for kubelock - remote project.yaml and consistently name env vars (E2E_*).